addons/api/addon/adapters/application.js

```diff
@@ -120,6 +120,10 @@ export default class ApplicationAdapter extends RESTAdapter.extend(
 
     // Run this loop as long as the response_type is delta,
     // which indicates that there are more items in the list
```

Comment:
This comment goes for the do loop, we should add another comment explaining why this hack is needed

Reply:
Good catch. Updated.